src/math/crypto/curve/ec_point.zig

```diff
@@ -19,6 +19,15 @@ pub const ProjectivePoint = struct {
         };
     }
 
+    fn identity() Self {
+        return .{
+            .x = Felt252.zero(),
+            .y = Felt252.zero(),
+            .z = Felt252.one(),
```

Comment:
Here you don't need this line as long as this value is already provided as default, let us also put `Felt252.zero()` as default in structure construction for `x` and `y` coordinates, so that you will also be able to remove these two lines from the function.